### DISCUSSION TITLE
Question about the Klein-Gordon equation demo

### DISCUSSION THREAD

**Discussion by ghost** [lululxvi/deepxde#1169] [Q&A]
I was studying the demo featuring the Klein Gordon equation in the DeepXDE website: https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/klein.gordon.html#

The reference solution is y = x*cost, so y = -1 when x = -1 and t = 0.

However, the prediction obtained by `model.predict` at x = -1 and t = 0 is y_pred = -0.77..., which is significantly different

from the real solution y = -1. Using a larger network size and increasing residual points on the domain, boundary, and initial conditions didn't seem to help.

Am I misunderstanding something here, or is PINN just poor at accurately predicting the value of Klein-Gordon eq at x = -1 and t = 0?

**Comment by lululxvi** [comment]
How is accuracy in the entire domain?
